src/Cassandra/UdtMap.cs

```diff
@@ -83,7 +84,9 @@ public abstract class UdtMap
         protected readonly Dictionary<PropertyInfo, string> _propertyToFieldName;
         // ReSharper enable InconsistentNaming
         private Serializer _serializer;
-        protected const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+        public const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+
+        public TypeConverter TypeConverter { get; private set; }
```

Comment:
Initially, it would be best not to expose the type converter in `UdtMap` class.
`TypeConverter` is an abstract class exposed on a higher namespace: `Cassandra.Mapping.TypeConversion` that was exposed for the Mapper and Linq components of the driver.

I think that we can internally reuse it, use a single instance for all `UdtMap` and if users in the future start asking for custom conversions outside of the default type converter (enums, collections, dates, ... are already covered), we could expose an interface that would be valid for both (udts and Mapper/Linq component).